tests/test_pipelines_automatic_speech_recognition.py

```diff
@@ -215,3 +215,20 @@ def test_xls_r_from_en(self):
         filename = ds[40]["file"]
         output = speech_recognizer(filename)
         self.assertEqual(output, {"text": "Ein Mann sagte zu dem Universum, Sir, ich bin da."})
+
+    @slow
+    @require_torch
+    @require_torchaudio
+    def test_wav2vec2_2_bart(self):
+        speech_recognizer = pipeline(
+            task="automatic-speech-recognition",
+            model="patrickvonplaten/wav2vec2-2-bart-base",
+            feature_extractor="patrickvonplaten/wav2vec2-2-bart-base",
+            tokenizer=AutoTokenizer.from_pretrained("patrickvonplaten/wav2vec2-2-bart-base"),
+            framework="pt",
+        )
+
+        ds = load_dataset("hf-internal-testing/librispeech_asr_dummy", "clean", split="validation").sort("id")
```

Comment:
Is the sort necessary ?

Reply:
yeah since it different versions datasets might break the order

Reply:
This seems like a very odd thing in `datasets` that order could be messed up by subsequent versions...

Reply:
@anton-l and I have experienced this now a couple of times so tat we'll just make sure this way

Reply:
Good to know !